README.md

```diff
@@ -6,6 +6,8 @@ SqlPackage.exe is a command line interface to DacFx and is available for Windows
 
 Microsoft.Build.Sql (preview) is a [.NET project SDK](https://docs.microsoft.com/dotnet/core/project-sdk/overview) for SQL projects, compiling T-SQL code to a data-tier application package (dacpac).  The SQL project SDK is available via NuGet ([https://www.nuget.org/packages/Microsoft.Build.Sql/](https://www.nuget.org/packages/Microsoft.Build.Sql/)) and in this [repository](/src/Microsoft.Build.Sql/README.md).
 
+Microsoft.Build.Sql.Templates (preview) is a set of [.NET project templates](https://learn.microsoft.com/dotnet/core/tools/custom-templates) for SQL projects.  The SQL project SDK templates are available via NuGet ([https://www.nuget.org/packages/Microsoft.Build.Sql.Templates/](https://www.nuget.org/packages/Microsoft.Build.Sql.Templates/)).
```

Comment:
This paragraph would be more concisely written as below. (It also included the wording from my previous comment).

[Microsoft.Build.Sql.Templates (preview)](https://www.nuget.org/packages/Microsoft.Build.Sql.Templates/) is a set of [.NET project templates](https://learn.microsoft.com/dotnet/core/tools/custom-templates) for [Microsoft.Build.Sql](https://www.nuget.org/packages/Microsoft.Build.Sql).